metabase/metabase
# Close collection sidebar on collection click in mobile

⚠️ Please review https://github.com/metabase/metabase/pull/16896 first

<hr />

## To test manually

1. Be in _mobile_ mode in the browser
2. Go to http://localhost:3000/collection/root
3. Click on the burger icon to show collections sidebar
4. Click on any collection inside the sidebar

This should:

1. close the sidebar
2. display the clicked collection
3. update the url on the url bar

### Demo

https://user-images.githubusercontent.com/380816/127412827-71907560-5024-4dda-b303-1ce38ad4380c.mov
